test/Streamly/Test/Data/Array/CommonCore.hs

```diff
@@ -0,0 +1,104 @@
+-- Coverage build takes too long with default number of tests
```

Comment:
Why do we need to copy this file? Looks like it only uses SerialT which is actually Stream now. Can't we just use the original file itself?